### PR TITLE
add bit on remote reverse proxy

### PR DESCRIPTION
**Pull request by hanscees**
The current documentation assumes the reverse proxy is on the same docker host. If it is not, the setup and tests are different. An external website with exmaples is referenced.

### CODE REVIEW

**Comment by hanscees** [comment]
You are of course also welcome to copy-paste the images and text to your own text.

**Review by Zoey2936 — CHANGES_REQUESTED** [review]
I don't understand this change, I think you can also mention your pdf in a disscusion, but everything is already documented, just use the given configs and adjust them, for diffrent hosts see https://github.com/nextcloud/all-in-one/blob/0aa4f63244ac761489dc4a2197508548e214a4ff/reverse-proxy.md?plain=1#L26

**Comment by Zoey2936** [comment]
Also I don't understand why you disable automatic start of the aio container after a reboot

**Comment by hanscees** [comment]
The documentation you have on reverse proxy is lacking in several ways:
- it does not describe the purpose of all the steps and does not explain what is achieved. Therefore it is very hard to follow if you dont know how the nextcloud-aio containers work together.
- it does not describe a remote proxy. 
- it is all tekst and no figures (graphical thinkers are very common)
and Therefore you have a constant stream of people that do not get it working.

I couldnt understand it at all.

**Comment by Zoey2936** [comment]
1.1. See https://github.com/nextcloud/all-in-one#how-does-it-work
1.2. I don't understand what needs to be explained you configure a reverse proxy and then start the mastercontainer
2. It is explained just use the ip you need instead of localhost
3. What do you mean with graphical? Pictures makes sense when showing structures, but not for config examples

**Comment by hanscees** [comment]
I am a bit confused. 
On your page reverse proxy page it sais: 
**Note: The maintainers of AIO noticed that this documentation is not easy. All contributions that improve this are very welcome!**

So I offer extra explanation and you say you dont need extra explanation. 
Obviously you know how it all works, but your readers do not and the explanation is not doing its job: explaining to other people how to configure things.

**Comment by Kirk1984** [comment]
To be frank, the mindmap is really overwhelming. I can't see how it should make it easier to understand. And your example also has IPs that need to be changed for each person. Just like localhost/127.0.0.1.

**Comment by Zoey2936** [comment]
> I am a bit confused. On your page reverse proxy page it sais: **Note: The maintainers of AIO noticed that this documentation is not easy. All contributions that improve this are very welcome!**
> 
> So I offer extra explanation and you say you dont need extra explanation. Obviously you know how it all works, but your readers do not and the explanation is not doing its job: explaining to other people how to configure things.

yes, the documentation is complicated written, also thanks for the PR, but I don't see an advantage in a mindmap for this

**Comment by hanscees** [comment]
I was hoping to help after two days wading through your documentation, and not finding why nextcloud did not work.  Apparently I am not able to help you. 

Perhaps you should not ask people to help you if you won't accept it? . I write for admin journal to help people get things going, but I guess you people know better.
It was very frustrating trying to help you. Nevertheless I wish you all the best.

**Comment by Zoey2936** [comment]
You don't need to close this, I don't decide what will be merged and what not, I just give my thoughts on this
